gradio/blocks.py

```diff
@@ -997,6 +1026,10 @@ def close(self, verbose: bool = True) -> None:
         Closes the Interface that was launched and frees the port.
         """
         try:
+            from gradio.event_queue import Queue
```

Comment:
why importing here when we already have a reference to event_queue imported at the top?